torch/_inductor/triton_ops/autotune.py

```diff
@@ -80,11 +79,13 @@ def _precompile_config(self, cfg: Config, warm_cache_only_with_cc: int):
             return
 
         torch.cuda.set_device(torch.cuda.current_device())
-
-        binary = triton.compile(
-            self.fn,
-            **compile_meta,
-        )
+        with torch.cuda.device(compile_meta["device"]):
```

Comment:
is the set_device still needed on L81 above?

Reply:
Right, that set_device is not needed, thanks!